src/Agent/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] changes
+## [9.3.0] changes
```

Comment:
Can you add the date (2022-01-04) next to the release number like previous releases, please?  Otherwise, looks good.